tests/test_workflow_fmriprep.py

```diff
@@ -87,29 +86,29 @@ def test_run_fmriprep(tmp_path, use_bids_filter, anat_only):
     )
     # fmt: off
     expected_cmd = [
-        'singularity', 'run', 
-            '-B', f'{bids_dir}:/data_dir', 
-            '-B', f'{fmriprep_dir}/output//fmriprep_home_{participant_id}/:/home/fmriprep', 
-            '--home', '/home/fmriprep', 
-            '--cleanenv', 
-            '-B', f'{fmriprep_dir}/output/:/output', 
-            '-B', f'{templateflow_dir}:/templateflow', 
-            '-B', f'{fmriprep_dir}:/work', 
-            '-B', f'{fs_dir}:/fsdir/', 
-                singularity_container, '/data_dir', '/output', 'participant', 
-                    '--participant-label', participant_id, 
-                    '-w', '/work', 
-                    '--output-spaces', 'MNI152NLin2009cAsym:res-2', 'anat', 'fsnative', 
-                    '--fs-subjects-dir', '/fsdir/', 
-                    '--skip_bids_validation', 
-                    '--bids-database-dir', '/work/first_run/bids_db/', 
-                    '--fs-license-file', '/fsdir/license.txt', 
-                    '--return-all-components', 
-                    '-v', 
-                    '--write-graph', 
-                    '--notrack', 
-                    '--omp-nthreads', '4', 
-                    '--nthreads', '8', 
```

Comment:
and that's why we use pre-commit...
otherwise your diff light up like a Xmas tree